File: monitoring-server-old/MonitoringServer/Models/MonitorParam.cs
using System;

namespace MonitoringService.Models
{
    /// <summary>
    /// Represents auxiliary instance that describes
    /// connections between monitors and tracked params.
    /// </summary>
    public class MonitorParam
    {
        /// <summary>
        /// Internal monitor identifier.
        /// </summary>
        public long MonitorId { get; set; }
        /// <summary>
        /// External param identifier.
        /// </summary>
        public Guid ParamUid { get; set; }
        /// <summary>
        /// Navigation property to monitor instance.
        /// </summary>
        public virtual Monitor Monitor { get; set; }
    }
}

File: monitoring-server/monitoring-service/src/main/scala/com/qwerfah/monitoring/models/MonitorParam.scala
package com.qwerfah.monitoring.models

import com.qwerfah.common.Uid

/** Represent connection between monitor and tracked equipment instance params.
  * @param monitorUid
  *   Monitor identifier.
  * @param paramUid
  *   Tracked param identifier.
  */
final case class MonitorParam(
  monitorUid: Uid,
  paramUid: Uid,
  isDeleted: Boolean
)
